src/app/component/SliderWidget/SliderWidget.container.js

```diff
@@ -70,6 +71,12 @@ export class SliderWidgetContainer extends DataContainer {
     }
 
     render() {
+        const { slider: { slideSpeed } } = this.state;
+
+        if (slideSpeed === null) {
```

Comment:
At first container is sending default values into component and only after second request sending correct values. But auto-scroll functionality is implemented in component componentDidMount() function, that at first starts with default values from container witch is not correct. So this IF is not allowing to send unupdated values (default) into component.